Bookish/Models/EditBooks.cs

```diff
@@ -2,49 +2,41 @@
 
 public class EditBooks
 {
-    public static void InsertBooks(string author, string title)
+    static LibraryContext context = new LibraryContext();
+    public static List<BookModel> GetBookList()
     {
-        using var context = new LibraryContext();
-        var book = new BookModel
-        {
-            Author = author,
-            Title = title
-        };
-        context.Books.Add(book);
+        var books = context.Books.ToList();
+        return books;
+    }
+    
+    public static BookModel GetBookFromId(int id)
+    {
+        BookModel book = context.Books.Single(b =>b.Id == id);
+        return book;
+    }
+        
+    public static void InsertBooks(BookModel newbook)

```

Comment:
If these methods (`InsertBooks`, `AlterBooks`, etc.) act on a single book, I would name them as such - `InsertBook` and `AlterBook` - to be clear